bin/dummy_mentor

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+questionnaire=$1
+default_page=index.html
+last_page=final.html
+questionnaire_page=questionnaire.html
+
+cd public/dummy
+
+rm -rf mentor
+wget \
+		 --save-cookies /tmp/cookies.txt \
+     --keep-session-cookies \
+     --recursive \
+     --no-clobber \
+     --page-requisites \
+     --html-extension \
+     --domains vsv.io \
+     --restrict-file-names=unix \
+     --no-parent \
+		 -E -H -k -K -p \
+     -r -e robots=off \
+     -U mozilla \
+    http://vsv.io\?q\=$questionnaire
+
+mv vsv.io mentor
+mv mentor/$default_page?q=$questionnaire.html mentor/$default_page
+rm mentor/*.orig
+rm mentor/**/*.orig
+
+cd mentor
+cp $default_page $last_page
+
+# Update the asset urls
+gsed -i "s/assets/\/dummy\/mentor\/assets/g" *.html
+#gsed -i "s/assets/\/dummy\/mentor\/assets/g" questionnaire/*.html
```

Comment:
Comment kan weg? Of hebben we die later nog nodig

Reply:
done